<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Deepbionics</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.15.1/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet" />
        <link href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i" rel="stylesheet" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
            <div class="container">
                <!--<a class="navbar-brand js-scroll-trigger" href="#page-top">Start Bootstrap</a>-->
                <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
                    Menu
                    <i class="fas fa-bars"></i>
                </button>
                <div class="collapse navbar-collapse" id="navbarResponsive">
                    <ul class="navbar-nav ml-auto">
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="index">Home</a></li>
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="projects">Projects</a></li>
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="rpackages">R Packages</a></li>
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="lectures">Lectures</a></li>
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="publications">Publications</a></li>
                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="contactme">Contact Me</a></li>
                    </ul>
                </div>
            </div>
        </nav>
        <!-- Markdown Part-->
        <header class="masthead">
            <div class="container d-flex h-100 align-items-center">
                <div class="mx-auto text-center">
                    <h1 class="mx-auto my-0 text-uppercase">Data Science</h1>
                    <h2 class="text-white-50 mx-auto mt-2 mb-5">Projects.</h2>
                    <a class="btn btn-primary js-scroll-trigger" href="#aboutProjects">Learn More</a>
                </div>
            </div>
        </header>
        <!-- Projects-->
        <section class="projects-section bg-light" id="aboutProjects">
             
            
          <section class="my-plain-text-section">
               
              The explanations and reasons for decisions made by algorithms concerning the health and/or treatment options for patients are firmly demanded. This calls for systems that produce human-understandable knowledge and decision proposals out of the data such that domain experts can base their decision on these systems. There are different names for such systems: symbolic, knowledge-based, expert systems or recently “explainable AI”  systems.
              Current explainable machine learning systems have either specific assumptions about the data or often do not follow the Grice's maxims, i.e., the explanations are not meaningful and relevant to domain expert <a href="https://doi.org/10.3390/make3010009">[Thrun et al., 2021]</a>. Using the already researched insights about structures in data and swarm intelligence  <a href="https://doi.org/10.1016/j.artint.2020.103237">[Thrun/Ultsch, 2021]</a>, an explainable machine learning system (XAI), allowing for user interaction and understandable both upon the Grice’s maxims and from the perspective of domain experts is curently researched. 
           </section>
            
            
          <section class="my-plain-text-section">
            The project is about the creation of a reference data analysis platform for immunophenotyping of B-cell non-Hodgkin lymphomas (B-NHL) using custom-fit artificial intelligence methods that justify their decision in such a way that a domain expert is able to review the decision. The goal is to automate usual simple cases (e.g., chronic lymphocytic leukemia) so that the domain expert can focus on complex cases. The group of B-NHL consists of various entities (diagnosis). The platform will aid the users in the classification of peripheral blood and bone marrow (BM) samples into different B-NHL entities. Due to the European legal situation, the suggested classification of a sample by the AI serves as a reference only and is provided with a human-understandable explanation after the physician gives an initial diagnosis. The project is called “Phenotyping Lymphomas with an Artificial Intelligence toolbox” (PLAIT). Some challenges are the proof-of-concept and building of a prototype that works on data from different cytometer units as well as data from other flow cytometer types (e.g., data standardization), and exploring the transferability on other panels, as well as the integration of another clinical center in PLAIT.
          </section>
        
            
            <section class="my-plain-text-section">
              The next project is about the tumor grouping as a new modality of transcriptome-based personalized medicine. Algorithms of computational intelligence are applied to NGS gene expressions to obtain new knowledge from the totality of tumor genome data which goes beyond state-of-the-art targeted screening for mutations in tumor genes. Algorithms based on swarm intelligence will be used to process the high-dimensional data into group structures. In published preliminary projects, I have been able to show on microarray gene expression data that the method is in principle applicable to tumor biology datasets. The method will now be adapted for large NGS tumor transcriptome datasets. More accurate stratification (subgroup analysis) will be performed and correlated with known clinical and tumor genetic factors. The goal is to create a platform for data-driven classification of individual tumor transcriptomes into an optimal prognostic and diagnostic context.
            </section>
            
            Exemplary Finished Projects
            
                <section class="my-plain-text-section">
              Clustering is an important task in knowledge discovery with the goal of finding groups of similar data points in a dataset. Currently, there are many different approaches to clustering, including methods to incorporate user decisions into the clustering process through various displays of structures in the data. Some of these interactive approaches fall into the category of visual analytics and emphasize the power of displays of structures in data to help find clusters interactively in various types of datasets or to verify the results of clustering algorithms. The interactive projection-based clustering (IPBC) is an open source and parameter-free tool using user input for an interactive display of structures in data in 2.5D and 3D based on the user’s choice of a projection method. IPBC uses cluster high-dimensional data either interactively or automatically.
            </section>
            
            One aim of data mining is the identification of interesting structures in data. For better analytical results, the basic properties of an empirical distribution, such as skewness and eventual clipping, i.e. hard limits in value ranges, need to be assessed. Of particular interest is the question of whether the data originate from one process or contain subsets related to different states of the data producing process. Data visualization tools should deliver a clear picture of the univariate probability density distribution (PDF) for each feature. Visualization tools for PDFs typically use kernel density estimates and include both the classical histogram, as well as the modern tools like ridgeline plots, bean plots and violin plots.
If density estimation parameters remain in a default setting, conventional methods pose several problems when visualizing the PDF of uniform, multimodal, skewed distributions and distributions with clipped data,  For that reason, a new visualization tool called the mirrored density plot (MD plot), which is specifically designed to discover interesting structures in continuous features, is proposed. The MD plot does not require adjusting any parameters of density estimation, which is what may make the use of this plot compelling particularly to non-experts.

                     <section class="my-plain-text-section">
            
            </section>
        </section>
        <!-- Contact-->
        <section class="contact-section bg-black">
            <div class="container">
                <!--
                <div class="row">
                    <div class="col-md-4 mb-3 mb-md-0">
                        <div class="card py-4 h-100">
                            <div class="card-body text-center">
                                <i class="fas fa-map-marked-alt text-primary mb-2"></i>
                                <h4 class="text-uppercase m-0">Address</h4>
                                <hr class="my-4" />
                                <div class="small text-black-50">4923 Market Street, Orlando FL</div>
                            </div>
                        </div>
                    </div>
                    <div class="col-md-4 mb-3 mb-md-0">
                        <div class="card py-4 h-100">
                            <div class="card-body text-center">
                                <i class="fas fa-envelope text-primary mb-2"></i>
                                <h4 class="text-uppercase m-0">Email</h4>
                                <hr class="my-4" />
                                <div class="small text-black-50"><a href="#!">hello@yourdomain.com</a></div>
                            </div>
                        </div>
                    </div>
                    <div class="col-md-4 mb-3 mb-md-0">
                        <div class="card py-4 h-100">
                            <div class="card-body text-center">
                                <i class="fas fa-mobile-alt text-primary mb-2"></i>
                                <h4 class="text-uppercase m-0">Phone</h4>
                                <hr class="my-4" />
                                <div class="small text-black-50">+1 (555) 902-8832</div>
                            </div>
                        </div>
                    </div>
                </div>-->
                <div class="social d-flex justify-content-center">
                    <a class="mx-2" href="https://github.com/Mthrun"><i class="fab fa-github"></i></a>
                    <a class="mx-2" href="https://www.researchgate.net/scientific-contributions/Michael-Christoph-Thrun-2183531503"><i class="fab fa-researchgate"></i></a>
                    <a class="mx-2" href="https://scholar.google.de/citations?hl=de&user=LyA4BVEAAAAJ"><i class="fab fa-google"></i></a>
                    <a class="mx-2" href="mailto:info@deepbionics.org"><i class="fab fa-envelope"></i></a>
                </div>
            </div>
        </section>
        <!-- Footer-->
        <footer class="footer bg-black small text-center text-white-50"><div class="container">Copyright © Deepbionics 2018 <a href="impressum">Impressum</a> </div></footer>
        <!-- Bootstrap core JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Third party plugin JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
